iot-e2e-tests/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +0,0 @@
-distributionBase=GRADLE_USER_HOME
-distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.8.3-bin.zip
-zipStoreBase=GRADLE_USER_HOME
-zipStorePath=wrapper/dists
```

Comment:
These fields were being ignored anyways, so I'm deleting them to avoid confusion about what version of gradle we use